src/lib.rs

```diff
@@ -79,6 +80,9 @@ mod server;
 mod state;
 mod toolchain;
 
+/// A container to store global customizations initialized upon launch.
+pub(crate) static TRICKS: OnceLock<Tricks> = OnceLock::new();
```

Comment:
```suggestion
static TRICKS: OnceLock<Tricks> = OnceLock::new();
```

It's defined in the root module, so it's accessible everywhere in this crate.

Reply:
Done